src/main/kotlin/io/github/dockyardmc/ServerMetrics.kt

```diff
@@ -43,10 +42,7 @@ object ServerMetrics {
             val diff = Instant.now().toEpochMilli() - timeSinceLastTick.toEpochMilli()
             averages.add(diff)
             timeSinceLastTick = Instant.now()
-            millisecondsPerTick = averages.average().truncate(1).toDouble()
-            if(millisecondsPerTick < 50) millisecondsPerTick = 50.0
-
-
+            millisecondsPerTick = Math.round(averages.average() * 10) / 10.0
```

Comment:
I'd be better to fix the core issue which is the truncate function cause it's used in more places

Reply:
(it is custom extension function I wrote)

Reply:
should .truncate still return a String, just with American localization, or should it return a double?

Reply:
Yeah string, it was mainly meant to be used in text

Reply:
Alrighty! I made it return with `Locale.ROOT`, which I'm pretty sure does the thing. Also made a `.round(decimals: Int)` func which imo should work better in the case of mspt rounding